### PR TITLE
Megamenu - dynamic contact us link

### LINKED ISSUE #1534: 🐛 New Mega Menu - Call us link
**Describe the bug**

Hi, 
this issue was previously reported and fixed in this issue:
https://github.com/SSWConsulting/SSW.Website/issues/1300

- [ ] Can you please ensure that the functionality of "Contact us" on desktop and "Call us" on mobile are maintained in the new version of the mega menu

Screenshot 2023-10-13 at 10 49 39 am

### PR DESCRIPTION
**Pull request by wicksipedia**
Made the contact/call us link work like the old megamenu
Tailwind - use default screen sizes
Megamenu - mobile layout - allow back navigation when clicking on menu header (larger target)
Chatbase - co-locate environment variable with component

- Affected routes: `*`

- Fixed #1534

- [x] Include done video or screenshots

CleanShot 2023-10-20 at 16 16 26

CleanShot 2023-10-20 at 15 52 38

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -3,13 +3,13 @@ const plugin = require("tailwindcss/plugin");
 
 /** @type {import("tailwindcss").Config} */
 module.exports = {
-  mode: "jit",
+  // mode: "jit",
```
Comment (bradystroud) [inline tailwind.config.js:6]: Why was JIT turned off?

File: components/megamenu/MegaMenuLayout/mega-menu-layout.tsx [hunk 2]
```diff
@@ -32,10 +32,11 @@ const MegaMenuLayout: React.FC<MegaMenuWrapperProps> = ({
               passHref
               className="unstyled flex items-center gap-1 whitespace-nowrap"
             >
-              <div className="flex items-center justify-center">
+              {/* eslint-disable-next-line tailwindcss/no-arbitrary-value */}
+              <div className="flex min-w-[4rem] max-w-[14rem] items-center justify-center">
```
Comment (bradystroud) [inline components/megamenu/MegaMenuLayout/mega-menu-layout.tsx:36]: Assuming this is planned tech debt (I know there are lots of refactors coming to this menu)

**Review by bradystroud — CHANGES_REQUESTED** [review]

File: tailwind.config.js (hunk 1 above)
Comment (wicksipedia) [reply tailwind.config.js:6]: jit is the default
https://tailwindcss.com/docs/upgrade-guide#migrating-to-the-jit-engine

File: components/megamenu/MegaMenuLayout/mega-menu-layout.tsx (hunk 2 above)
Comment (wicksipedia) [reply components/megamenu/MegaMenuLayout/mega-menu-layout.tsx:36]: yeah just trying to make it layout nicely

File: tailwind.config.js (hunk 1 above)
Comment (bradystroud) [reply tailwind.config.js:6]: Can we delete the line then?

**Review by bradystroud — APPROVED** [review]
LGTM

**Pull request merged**